src/blister.c

```diff
@@ -1568,7 +1568,7 @@ Obj FuncLIST_BLIST (
     while ( LEN_LIST( list ) != LEN_BLIST( blist ) ) {
         blist = ErrorReturnObj(
             "ListBlist: <blist> must have the same length as <list> (%d)",
-            LEN_PLIST( list ), 0L,
+            LEN_LIST( list ), 0L,
```

Comment:
I find the commit message unclear. How about "Stop calling LEN_PLIST on generic lists" or "Avoid ..." -- when you say "fix", that could also mean that you are changing things so that this actually becomes valid.